I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 34, in script
    "*buzz buzz*"
  File "renpy/common/000window.rpy", line 98, in _window_auto_callback
    _window_show()
  File "renpy/common/000window.rpy", line 60, in _window_show
    renpy.with_statement(trans)
IOError: Couldn't find file 'vibrate.mp3'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "/Users/seanbernhard15/Desktop/renpy-6.99.13-sdk/renpy/bootstrap.py", line 305, in bootstrap
    renpy.main.main()
  File "/Users/seanbernhard15/Desktop/renpy-6.99.13-sdk/renpy/main.py", line 499, in main
    run(restart)
  File "/Users/seanbernhard15/Desktop/renpy-6.99.13-sdk/renpy/main.py", line 147, in run
    renpy.execution.run_context(True)
  File "/Users/seanbernhard15/Desktop/renpy-6.99.13-sdk/renpy/execution.py", line 795, in run_context
    context.run()
  File "game/script.rpy", line 34, in script
    "*buzz buzz*"
  File "/Users/seanbernhard15/Desktop/renpy-6.99.13-sdk/renpy/ast.py", line 590, in execute
    statement_name("say")
  File "/Users/seanbernhard15/Desktop/renpy-6.99.13-sdk/renpy/ast.py", line 43, in statement_name
    i(name)
  File "renpy/common/000window.rpy", line 98, in _window_auto_callback
    _window_show()
  File "renpy/common/000window.rpy", line 60, in _window_show
    renpy.with_statement(trans)
  File "/Users/seanbernhard15/Desktop/renpy-6.99.13-sdk/renpy/exports.py", line 1406, in with_statement
    return renpy.game.interface.do_with(trans, paired, clear=clear)
  File "/Users/seanbernhard15/Desktop/renpy-6.99.13-sdk/renpy/display/core.py", line 2123, in do_with
    clear=clear)
  File "/Users/seanbernhard15/Desktop/renpy-6.99.13-sdk/renpy/display/core.py", line 2553, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, **kwargs)
  File "/Users/seanbernhard15/Desktop/renpy-6.99.13-sdk/renpy/display/core.py", line 3104, in interact_core
    renpy.audio.audio.periodic()
  File "/Users/seanbernhard15/Desktop/renpy-6.99.13-sdk/renpy/audio/audio.py", line 960, in periodic_thread_main
    periodic_pass()
  File "/Users/seanbernhard15/Desktop/renpy-6.99.13-sdk/renpy/audio/audio.py", line 917, in periodic_pass
    c.periodic()
  File "/Users/seanbernhard15/Desktop/renpy-6.99.13-sdk/renpy/audio/audio.py", line 409, in periodic
    topf = load(self.file_prefix + filename + self.file_suffix)
  File "/Users/seanbernhard15/Desktop/renpy-6.99.13-sdk/renpy/audio/audio.py", line 69, in load
    rv = renpy.loader.load(fn)
  File "/Users/seanbernhard15/Desktop/renpy-6.99.13-sdk/renpy/loader.py", line 548, in load
    raise IOError("Couldn't find file '%s'." % name)
IOError: Couldn't find file 'vibrate.mp3'.

Darwin-15.6.0-x86_64-i386-64bit
Ren'Py 6.99.13.2919
A Day in the Death 1.0
